src/index.css

```diff
@@ -1,13 +1,3 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```

Comment:
This was the font for the whole UI, right @neuralaidtu , why did you remove this?